linkis-commons/linkis-storage/src/main/scala/org/apache/linkis/storage/utils/StorageConfiguration.scala

```diff
@@ -83,4 +83,15 @@ object StorageConfiguration {
   val FS_CHECKSUM_DISBALE =
     CommonVars[java.lang.Boolean]("linkis.fs.hdfs.impl.disable.checksum", false)
 
+  val S3_ACCESS_KEY = CommonVars("wds.linkis.storage.s3.access.key", "")
```

Comment:
need to remove wds prefix

Reply:
Thanks for your advice.I've dropped the prefix.